### DISCUSSION TITLE
2023-04-21 demo

### DISCUSSION THREAD

**Discussion by ngjaying** [lf-edge/ekuiper#1801] [Demo]
Recording: https://zoom.us/rec/share/Qoulee565Zg96SjP1hwrj6owPj9q2Q09wyBuE9OxL4itTO7jpcGxD5OY2ILxC_uD.qi08n65VODl_PJAY

- Demo: Access external state in redis @superrxan 
- Demo: How to run examples in cookbook @ngjaying 
- Design discussion: unnest array column to multiple rows @Yisaer 

Join Zoom Meeting:  Apr 21st 10:30AM UTC+8

https://zoom.us/j/95097577087?pwd=azZaOXpXWmFoOXpqK293RFp0N1pydz09

Meeting ID: 950 9757 7087

Passcode: 998285

**Comment by carlclone** [comment]
link for the unnest design in video : https://lwghun58250.feishu.cn/docx/VUbYdXOyjoTAv7xsUpocrfQCnRd
